app/controllers/orga/teams_controller.rb

```diff
@@ -48,7 +47,7 @@ def create
   def update
     @conferences = conference_list
     respond_to do |format|
-      if @team.update_attributes(team_params)
+      if @team.update_attributes(team_params) && terms_accepted?
```

Comment:
I would prefer to use an [acceptance](http://guides.rubyonrails.org/active_record_validations.html#acceptance) validation here instead. I personally find it way clearer, since it does not leak any validation code from the model into the controller.

It works like this:

1. You define a validation on the model:
```ruby
class ConferencePreference
  validates :terms_of_ticket, acceptance: true
  validates :terms_of_travel, acceptance: true
end
```
☝️  note that these two attributes **are not** backed by a DB column

2. Add the necessary field to the form:
```ruby
= f.simple_fields_for :conference_preference do |c|
  = c.input :first_conference_id, collection: @conferences.group_by(&:region)
  = c.input :second_conference_id, collection: @conferences.group_by(&:region)
  = c.input :comment
  = c.input :terms_of_ticket, as: :checkbox
  = c.input :terms_of_travel, as: :checkbox
  # etc.
```

3. And then simply permit it in the controller